src/rtc.rs

```diff
@@ -70,8 +70,24 @@ impl Rtc {
         })
     }
 
-    /// Set the current rtc value to the specified amount of seconds
-    pub fn set_seconds(&mut self, seconds: u32) {
+    /// Selects the frequency of the counter
```

Comment:
To me, this comment is a bit hard to understand without knowlege of the RTC because it's not clear what `counter` is. Perhaps change it to `Selects the frequency of the timer`, though that doesn't feel quite right to me either